keyboards/zeal60/readme.md

```diff
@@ -0,0 +1,16 @@
+Zeal60
+====
+
+[Zeal60](https://cdn.shopify.com/s/files/1/0490/7329/products/Zeal60.jpg)
+
+This is a 60% PCB with per-key RGB LEDs and supports ANSI, ISO, winkey/winkeyless bottom row, HHKB-layout (split right shift and backspace).
+
+Keyboard Maintainer: [Wilba](http://wilba.tech/) and on [github](https://github.com/Wilba6582)
+Hardware Supported: Zeal60 PCB Rev 0-3
+Hardware Availability: https://zealpc.net/collections/group-buy-pre-orders/products/zeal60rgb
+
+Make example for this keyboard (after setting up your build environment):
+
+    make zeal60:default
+
+See [build environment setup](https://docs.qmk.fm/build_environment_setup.html) then the [make instructions](https://docs.qmk.fm/make_instructions.html) for more information.
```

Comment:
Please change this line to the following:

`See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).`